compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELowerToUKernels.cpp

```diff
@@ -80,6 +80,38 @@ static FnNameAndDefAttrs getFnNameAndDefAttrs(RewriterBase &rewriter,
 /// ================== SAME UTILITIES AS IREE LLVMCPU ====================
 /// ======================================================================
 
+/// Utility to fetch the element type as string.
+static std::string typeToString(Type elemType) {
+  if (elemType.isSignlessInteger(8)) {
+    return "i8";
+  } else if (elemType.isSignlessInteger(32)) {
+    return "i32";
+  } else if (elemType.isBF16()) {
+    return "bf16";
+  } else if (elemType.isF32()) {
+    return "f32";
+  } else {
+    assert(false && "unsupported element type");
```

Comment:
```suggestion
    assert(false && "unimplemented element type in typeToString");
```
Just so that the person who hits this in the future doesn't feel too hopeless